packages/core/src/js/tracing/span.ts

```diff
@@ -104,7 +105,7 @@ export const startIdleSpan = (
     return new SentryNonRecordingSpan();
   }
 
-  getCurrentScope().setPropagationContext(generatePropagationContext());
+  getCurrentScope().setPropagationContext({ traceId: generateTraceId(), sampleRand: Math.random() });
```

Comment:
Q: Should we use Math.random() here?

Reply:
I think this should work given how `sampleRand` [is defined](https://github.com/getsentry/sentry-javascript/blob/317782c38206b775fcc5cdf42a2fc8c30f52c9ac/packages/core/src/types-hoist/tracing.ts#L23). It is [also used on the javascript sdk](https://github.com/search?q=repo%3Agetsentry%2Fsentry-javascript%20%22sampleRand%3A%20Math.random()%22&type=code).